bats/tests/preferences/verify-settings.bats

```diff
@@ -0,0 +1,53 @@
+load '../helpers/load'
+
+@test 'initial factory reset' {
+    factory_reset
+}
+
+@test 'start the app' {
+    start_container_engine
+    wait_for_container_engine
+}
+
+@test 'complain about windows-specific vm settings' {
+    skip_on_windows
+    run rdctl api /settings
+    assert_success
+    proxyEnabled=$(jq .experimental.virtualMachine.proxy.enabled <<<"$output")
+    settingsTemplateStart='{ "version": '$(get_setting .version)', "experimental": { "virtualMachine": { "proxy":'
+    settingsTemplateEnd='} } }'
```

Comment:
I think this is kind of hard to read. I would rather add a function

```shell
proxy_set() {
    local field=$1
    local value=$2

    run rdctl api settings -X PUT --body '{ "version": '$(get_setting .version)', "experimental": { "virtualMachine": { "proxy": { '$(json_string "$field")': $value}}}}'
    assert_failure
    assert_output --partial "$(printf $'Changing field "experimental.virtualMachine.proxy.%s" via the API isn\'t supported' $field)"
}
```

(Just hacked up in the web interface, so probably won't work as-is)

and then call it

```
@test ... {
    proxy_set enabled true
    proxy_set address "$(json_string "foo")"
    ...
}